packages/devops/scripts/deployment/checkoutLatest.sh

```diff
@@ -3,6 +3,7 @@
 # Get latest code and dependencies
 echo "Checking out ${BRANCH}, or dev if that doesn't exist"
 cd ${HOME_DIRECTORY}
+git remote prune origin
```

Comment:
@edmofro this should solve the issue where deleted branches are can still be checked out during deployment

Will test that in a feature instance